tests/ltp/patches/chown04.patch

```diff
@@ -0,0 +1,98 @@
+diff --git a/testcases/kernel/syscalls/chown/chown04.c b/testcases/kernel/syscalls/chown/chown04.c
+index 1f3ed412b..487aec16e 100644
+--- a/testcases/kernel/syscalls/chown/chown04.c
++++ b/testcases/kernel/syscalls/chown/chown04.c
+@@ -37,7 +37,8 @@
+  *   6) chown(2) returns -1 and sets errno to ENOENT if the specified file
+  *              does not exists.
+  */
+-
++/* Patch Description: Test cases were failing as there was not sufficient memory to use for test image creation for a loop device for test purpose.
++Solution: Test cases are modified to use /dev/vda the root file system device. In test accessing fault address is commented. This is a known issue in sgx that it causes enclave abort and is being tracked in Issue 169. Also commented testcase which checks mounted filesystem is readonly or not , it has a bug will be tracked under issue 189*/
```

Comment:
All sentences after the first don't make sense in the context of "Solution:". Please restructure this in a more understandable way, addressing each problem that the patch solves separately.